checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -160,6 +160,12 @@ def _check_suppression(self, record: Record, suppression: dict[str, Any]) -> boo
                         return True
             return False
 
+        elif type == 'LicenseType':
+            for license_type in suppression['licenseTypes']:
+                if record.vulnerability_details and record.vulnerability_details['license'] == license_type:
+                    return True
+            return False
```

Comment:
This code looks like this
https://docs.python.org/3/library/functions.html#any
WDYT about changing it? :)